modules/reference/pages/rpk/rpk-topic/rpk-topic-create.adoc

```diff
@@ -32,14 +25,13 @@ create topics.
 
 |-h, --help |- |Help for create.
 
-|-p, --partitions |int32 |Number of partitions to create per topic; -1
-defaults to the cluster's default_topic_partitions (default -1).
+|-p, --partitions |int32 |Number of partitions to create per topic; `-1`
+defaults to the cluster property xref:reference:cluster-properties.adoc#default_topic_partitions[`default_topic_partitions`] (default `-1`).
```

Comment:
```suggestion
defaults to the cluster property xref:reference:cluster-properties.adoc#default_topic_partitions[`default_topic_partitions`]. Default is `-1`.
```

Reply:
@Deflaimun: Could you please change all of defaults in the table from (default is x) to "Default is x." i.e., 2 separate sentences, similar to how we write defaults [here](https://docs.redpanda.com/current/manage/tiered-storage/#tiered-storage-configuration-properties).

Reply:
https://github.com/redpanda-data/documentation-private/issues/2286